src/SeqViz.tsx

```diff
@@ -380,7 +366,7 @@ export default class SeqViz extends React.Component<SeqVizProps, SeqVizState> {
    * Search for the query sequence in the part sequence, set in state.
    */
   search = (props: SeqVizProps, seq: string): { search: SearchResult[] } => {
-    const { onSearch, search: searchProp, translations, seqType } = props;
```

Comment:
The rest of the changes in this file are for the linter, I'm not sure why they didn't error before...